### PR TITLE
Rearrange Sections in learn-identity

### LINKED ISSUE #2604: Rearrange sections in Identity doc
Need to present the Registrars section coherently - https://wiki.polkadot.network/docs/learn-identity#registrars

Judgments
> Requesting Judgement
> Cancelling Judgement
Registrars
> Kusama
> Polkadot

### PR DESCRIPTION
**Pull request by salmad3**
resolves #2604

### CODE REVIEW

**Comment by salmad3** [comment]
Page screenshots will also be updated and formatted as per: https://github.com/w3f/polkadot-wiki/issues/2390

**Comment by DrW3RK** [comment]
I guess it is time for me to work on that issue! Will spend time tomorrow updating the screenshots showing latest UI

**Comment by DrW3RK** [comment]
Following today's conversation, I will do the UI imagery updates using a different PR to keep things separate. After @lamafab reviews,  please merge this

File: docs/learn/learn-identity.md [hunk 1]
```diff
@@ -152,6 +116,34 @@ for the `max_fee` put the maximum you're willing to pay for these confirmations.
...
+Once the registrar has confirmed the identity, a green checkmark should appear next to your account
+name with the appropriate confidence level:
+
+![A confirmed identity](../assets/identity/10.jpg)
+
+_Note that changing even a single field's value after you've been verified will un-verify your
+account and you will need to start the judgement process anew. However, you can still change fields
+while the judgement is going on - it's up to the registrar to keep an eye on the changes._
+
 ## Cancelling Judgements
```
Comment (lamafab) [inline docs/learn/learn-identity.md]: Should be `### Cancelling Judgement` (3x `#` and singular), no?

Comment (salmad3) [reply docs/learn/learn-identity.md]: Thanks, updated.

**Pull request merged**